ecosystem-tool-user-case.md

```diff
@@ -0,0 +1,39 @@
+---
+title: TiDB Ecosystem Tools Use Cases
+summary: Learn the common use cases of TiDB ecosystem tools and how to choose the tools.
+category: reference
+---
+
+# TiDB Ecosystem Tools Use Cases
+
+This document introduces the common use cases of TiDB ecosystem tools and how to choose the right tool for your scenario.
+
+## Import data from CSV to TiDB
+
+If you need to import the compatible CSV files exported by other tools to TiDB, use [TiDB Lightning](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md).
+
+## Import full data from MySQL/Aurora
+
+If you need to import full data from MySQL or Aurora, use [Dumpling](/export-or-backup-using-dumpling.md) first to export data as SQL dump files, and then use [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) to import data into the TiDB cluster.
+
+## Migrate data from MySQL/Aurora
+
+If you need to migrate both full data and incremental data from MySQL/Aurora, use [TiDB Data Migration](https://pingcap.com/docs-cn/tidb-data-migration/stable/overview/) (DM) to perform the full and incremental data migration.
```

Comment:
https://pingcap.com/docs/tidb-data-migration/stable/overview/

@ran-huang Please fix it.

Reply:
Fixed in https://github.com/pingcap/docs/pull/2796